src/widgets/histogram/histogram-widget-model.js

```diff
@@ -35,6 +36,7 @@ module.exports = WidgetModel.extend({
   _updateAutoStyle: function (m, style) {
     if (this.autoStyler) {
       this.autoStyler.updateColors(style);
+      this.autoStyler.opacity = getValue(style, 'auto_style.definition.color.opacity');
```

Comment:
That's a good one, it works different than the category widget, so it doesn't make sense in this case, let's change it :D